createEvents.mjs

```diff
@@ -4,7 +4,7 @@ import moment from 'moment';
 export default function createEvents(override = {}) {
    return {
     "@timestamp": moment().format("yyyy-MM-DDTHH:mm:ss.SSSSSSZ"),
-    criticality: faker.helpers.arrayElement(["NOT_IMPORTANT", "NORMAL", "IMPORTANT", "VERY_IMPORTANT",]),
```

Comment:
I don't think these should have been capitalised unless I am misunderstanding?

Reply:
I dont know. Nikita was the one who did the criticality assignment here. Maybe at one point they were capitalised and we just never changed them here? 
It's weird that it still works? Although I do remember seeing some issues where we'd see less assigned criticalities than expected so maybe this is the cause?

But yeah, should be fixed now with your changes